moto/sesv2/responses.py

```diff
@@ -0,0 +1,44 @@
+"""Handles incoming sesv2 requests, invokes methods, returns responses."""
+import json
+
+from moto.core.responses import BaseResponse
+from .models import sesv2_backends
+from ..ses.models import ses_backends
+from ..ses.responses import SEND_EMAIL_RESPONSE
+
+
+class SESV2Response(BaseResponse):
+    """Handler for SESV2 requests and responses."""
+
+    def __init__(self):
+        super().__init__(service_name="sesv2")
+
+    @property
+    def sesv2_backend(self):
+        """Return backend instance specific for this region."""
+        return sesv2_backends[self.current_account][self.region]
+
+    @property
+    def sesv1_backend(self):
+        """Return backend instance specific for this region."""
+        return ses_backends[self.current_account][self.region]
+
+    def send_email(self):
+        """Piggy back on functionality from v1 mostly"""
+
+        # parsing of these params is nasty, hopefully there is a tidier way
+        params = json.loads(list(dict(self.querystring.items()).keys())[0])
+        from_email_address = params.get("FromEmailAddress")
+        destination = params.get("Destination")
+        content = params.get("Content")
+
+        message = self.sesv1_backend.send_email(
```

Comment:
This should really call the `sesv2_backend.send_email` - in there, we can then call `sesv1.backend`.
That way we have a clean separation of IO (in responses) and logic (in models). It also ensures that the method is picked up in the implementation coverage script.